src/test/javascript/jasmine.js

```diff
@@ -1,22 +1,15 @@
 const Jasmine = require('jasmine');
 const reporter = require('./lib/ddt_reporter');
```

Comment:
this file just got cleaned up a little, though the only behavior difference is it will now run the integration tests as the default behavior